pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -0,0 +1,186 @@
+				return fmt.Errorf("invalid cluster %s", choice)
+			}
+			cl, ok := clusterByNames[choice]
+			if !ok {
+				return fmt.Errorf("cannot find a cluster id for the cluster name %s", clusterName)
+			}
+			p.ClusterID = cl.MustId()
+			return nil
+		})
+
+	var templateName string
+	var templateByNames map[string]*ovirtsdk4.Template = make(map[string]*ovirtsdk4.Template)
+	var templateNames []string
+
+	templateList, err := systemService.TemplatesService().List().Search("cluster=" + clusterName).Send()
+	if err != nil {
+		return nil, err
+	}
+	templates, ok := templateList.Templates()
+	if !ok {
+		return nil, fmt.Errorf("could not fetch the VM templates of cluster %s", clusterName)
+	}
+	for _, tmpl := range templates.Slice() {
+		templateByNames[tmpl.MustName()] = tmpl
+		templateNames = append(templateNames, tmpl.MustName())
+	}
+
+	err = survey.AskOne(&survey.Select{
+		Message: "Pick a VM template",
+		Help:    "The VM template will be used to create every node in the cluster",
+		Options: templateNames,
+	},
+		&templateName,
+		func(ans interface{}) error {
+			choice := ans.(string)
+			sort.Strings(templateNames)
+			i := sort.SearchStrings(templateNames, choice)
+			if i == len(templateNames) || templateNames[i] != choice {
+				return fmt.Errorf("invalid template %s", choice)
+			}
+			tmpl, _ := templateByNames[choice]
+			p.TemplateID = tmpl.MustId()
+			return nil
+		})
+
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter the internal API Virtual IP",
+				Help:    "Make sure the ip is not used by any party",
+				Default: "",
+			},
+			Validate: survey.ComposeValidators(survey.Required),
+		},
+	}, &p.APIVIP)
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter the internal DNS Virtual IP",
+				Help:    "Make sure the ip is not used by any party",
+				Default: "",
+			},
+			Validate: survey.ComposeValidators(survey.Required),
+		},
+	}, &p.DNSVIP)
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter the ingress IP ",
+				Help:    "Make sure the ip is not used by any party",
+				Default: "",
+			},
+			Validate: survey.ComposeValidators(survey.Required),
+		},
+	}, &p.IngressVIP)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &p, nil
+}
+
+func askCredentials() (ovirt.OvirtConfig, error) {
+	c := ovirt.OvirtConfig{}
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter oVirt's api endpoint URL",
+				Help:    "oVirt engine api url",
+				Default: "https://ovirt-engine-fqdn/ovirt-engine/api",
+			},
+			Validate: survey.ComposeValidators(survey.Required),
+		},
+	}, &c.URL)
+
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter ovirt-engine username",
+				Help:    "The user must have permissions to create VMs, Disks, on the Storage Domain with the name matching the openshift cluster",
+				Default: "admin@internal",
```

Comment:
is this a valid default value? it looks like an example, in which case I don't think it should be a default and should perhaps be moved to Message

Reply:
Its a proper default value for the platform. It is recommended to create a dedicated user
for this setup, and I will add that to the help. In reality ovirt comes with a single, predefined super user.